<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="58-s07" name="FirefoxTest" tests="15" failures="0" timestamp="20 Aug 2018 10:48:16 GMT" time="1.152" errors="0">
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.CreateAccountTest">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &amp;apos;2.53.1&amp;apos;, revision: &amp;apos;a36b8b1&amp;apos;, time: &amp;apos;2016-06-30 17:37:03&amp;apos;
System info: host: &amp;apos;58-s07&amp;apos;, ip: &amp;apos;127.0.1.1&amp;apos;, os.name: &amp;apos;Linux&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;3.19.0-25-generic&amp;apos;, java.version: &amp;apos;1.7.0_151&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: '58-s07', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.19.0-25-generic', java.version: '1.7.0_151'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:158)
	at com.applect.meritnation.generic.BaseLib.preConditionWeb(BaseLib.java:123)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 10.0.2.83:6666 [/10.0.2.83] failed: Connection refused (Connection refused)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 26 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:580)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 39 more
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.CreateAccountTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.SearchTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.SearchTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.AsknAnswerTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.AsknAnswerTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.AsknAnswerTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.AsknAnswerTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="@BeforeMethod preConditionWeb" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod preConditionWeb -->
  <testcase name="@AfterMethod postCondition" time="0.0" classname="com.applect.meritnation.scripts.StudyTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.applect.meritnation.generic.BaseLib.postCondition(BaseLib.java:271)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)
	at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase name="TC_001_CreateAccountTest" time="0.005" classname="com.applect.meritnation.scripts.CreateAccountTest">
    <skipped/>
  </testcase> <!-- TC_001_CreateAccountTest -->
  <testcase name="TC_001_SearchTest" time="0.005" classname="com.applect.meritnation.scripts.SearchTest">
    <skipped/>
  </testcase> <!-- TC_001_SearchTest -->
  <testcase name="TC_001_LoginTest" time="0.002" classname="com.applect.meritnation.scripts.LoginTest">
    <skipped/>
  </testcase> <!-- TC_001_LoginTest -->
  <testcase name="TC_001_AskQuestionTest" time="0.003" classname="com.applect.meritnation.scripts.AsknAnswerTest">
    <skipped/>
  </testcase> <!-- TC_001_AskQuestionTest -->
  <testcase name="TC_003_VerifyUserProfileFromRecommendedQuestionsTest" time="0.003" classname="com.applect.meritnation.scripts.AsknAnswerTest">
    <skipped/>
  </testcase> <!-- TC_003_VerifyUserProfileFromRecommendedQuestionsTest -->
  <testcase name="TC_001_VerifyStudyPhysicsPageTest" time="0.003" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_001_VerifyStudyPhysicsPageTest -->
  <testcase name="TC_002_verifyStudyPhysicsTopicLiveClasssesTest" time="0.004" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_002_verifyStudyPhysicsTopicLiveClasssesTest -->
  <testcase name="TC_003_VerifyStudyPhysicsTopicConceptsTest" time="0.003" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_003_VerifyStudyPhysicsTopicConceptsTest -->
  <testcase name="TC_004_VerifyStudyPhysicsTopicPracticeTest" time="0.003" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_004_VerifyStudyPhysicsTopicPracticeTest -->
  <testcase name="TC_005_VerifyStudyPhysicsTopicAssignmentTest" time="0.006" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_005_VerifyStudyPhysicsTopicAssignmentTest -->
  <testcase name="TC_006_VerifyStudyPhysicsTopicTestsTest" time="0.004" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_006_VerifyStudyPhysicsTopicTestsTest -->
  <testcase name="TC_009_VerifyStudyPhysicsTopicRevisionNotesTest" time="0.004" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_009_VerifyStudyPhysicsTopicRevisionNotesTest -->
  <testcase name="TC_010_VerifyStudyPhysicsTopicBoardPaperQuestionsTest" time="0.003" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_010_VerifyStudyPhysicsTopicBoardPaperQuestionsTest -->
  <testcase name="TC_011_VerifyStudyPhysicsTopicFormativeAssesmentTest" time="0.003" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_011_VerifyStudyPhysicsTopicFormativeAssesmentTest -->
  <testcase name="TC_053_VerifyBoardPapterSolutionTest" time="0.003" classname="com.applect.meritnation.scripts.StudyTest">
    <skipped/>
  </testcase> <!-- TC_053_VerifyBoardPapterSolutionTest -->
</testsuite> <!-- FirefoxTest -->
